src/pages/AssetMapPage/AMDrawer.tsx

```diff
@@ -134,7 +134,7 @@ export const AMDrawer = () => {
           },
         ].concat(
           amEpochs.map(e => ({
-            label: e.labelGraph,
+            label: e.description ?? e.labelGraph,
```

Comment:
this looks like it might be a big ask, but is it possible to string-concat the description with the date range of the epoch?

Reply:
done :)